### PR TITLE
Bump to 0.11.1

### PR DESCRIPTION
**Pull request by gwarf**
Bump to 0.11.1.

### CODE REVIEW

**Comment by gwarf** [comment]
@brucellino can you please provide support/doc on managing the Zenodo/DOI update if needed?

File: .zenodo.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "license": "Apache-2.0",
     "description": "<ul>\n<li>Split OpenStack and OOI providers. (Enol Fernandez)</li>\n<li>Fix os_tpl identifier for OpenNebula with rOCCI. (Boris Parak)</li>\n<li>Clarify usage of project ID in configuration file. (Pablo Orviz)</li>\n<li>Review and update output to implement latest GLUE2.1 updates. (Enol Fernandez, Baptiste Grenier)</li>\n<li>Extract information from the GOCDB. (Enol Fernandez)</li>\n<li>Add debug switch. (Enol Fernandez)</li>\n<li>Clean up documentation. (Enol Fernandez)</li>\n<li>Use stevedore module to load code extensions. (Pablo Orviz)</li>\n<li>Document release management. (Baptiste Grenier)</li>\n</ul>",
```
Comment (enolfc) [inline .zenodo.json]: Shouldn't this be something like "Fixes #140: SSL utils fail when URL does not contain port."

File: debs/cloud-info-provider-opennebula/debian/changelog [hunk 2]
```diff
@@ -1,3 +1,10 @@
+python-cloud-info-provider-opennebula (0.11.1-1) xenial; urgency=medium
```
Comment (enolfc) [inline debs/cloud-info-provider-opennebula/debian/changelog]: Now that we have the stevedore stuff ready, we should think on moving this to separate packages that can be updated separately.

But, that's low priority at the moment.

File: .zenodo.json (hunk 1 above)
Comment (gwarf) [reply .zenodo.json]: Indeed, I intentionally didn't replace it to keep the previous changes, but should have added it. So will do it, thanks.

File: .zenodo.json [hunk 3]
```diff
@@ -1,82 +1,78 @@
...
+    {
+      "orcid": "0000-0002-2473-6405",
+      "name": "Pablo Orviz"
+    },
+    {
+      "affiliation": "Forschungszentrum J\u00fclich GmbH",
+      "name": "Bj\u00f6rn Hagemeier"
+    },
+    {
+      "name": "Rueben Diaz"
```
Comment (enolfc) [inline .zenodo.json]: s/Rueben/Ruben/

**Pull request merged**